uptane-standard.md

```diff
@@ -781,12 +782,13 @@ The ECU SHALL verify that the latest image matches the latest metadata as follow
 1. Load the latest Targets metadata file from the director.
 2. Find the Targets metadata associated with this ECU identifier.
 3. Check that the hardware identifier in the metadata matches the ECUs hardware identifier.
-4. Check that the release counter of the image in the previous metadata, if it exists, is less than or equal to the release counter in the latest metadata.
-5. If the image is encrypted, decrypt the image with a decryption key to be chosen as follows:
+4. Check that the image filename is valid for this ECU. This MAY be a comparison against a wildcard path, and restricts the ECUs a delegation applies to.
```

Comment:
I like the general idea here, but where would such a wildcard path come from? Are you expecting an ECU to know about some pre-configured path to match all filenames against?

Reply:
> I like the general idea here, but where would such a wildcard path come from? Are you expecting an ECU to know about some pre-configured path to match all filenames against?

Yes, the ECU would be configured with a path glob in the same way as it is configured with a Hardware Identifier. This is my understanding of this discussion with @JustinCappos at the last meeting --if we restrict delegations to a specific set of paths, then we need to make sure those paths have some meaning on the ECU. If we don't do that, then users will think they are getting security by restricting the paths in delegations, when in fact it offers no security benefit at all.